dependabot/update_lag_status.py

```diff
@@ -144,9 +144,10 @@ def update_modules(updated_readme, module_details_list):
 
             lag_status, color = get_lag_info(module[MODULE_NAME])
             lag_status += "%20days"
+            lag_status_link = "https://github.com/ballerina-platform/"+module[MODULE_NAME]+"/blob/main/gradle.properties"
```

Comment:
The default branch is not always main. The information is in extensions.json. Fix it ASAP.